src/app/onc-history-detail/onc-history-detail.component.ts

```diff
@@ -299,12 +294,12 @@ export class OncHistoryDetailComponent implements OnInit {
 
   openNoteModal( index: number ) {
     this.indexForNote = index;
-    this.note = this.oncHistory[ this.indexForNote ].oncHisNotes;
+    this.note = this.oncHistory[ this.indexForNote ].notes;
   }
 
   saveNote() {
-    this.oncHistory[ this.indexForNote ].oncHisNotes = this.note;
-    this.valueChanged( this.note, "oncHisNotes", this.indexForNote );
+    this.oncHistory[ this.indexForNote ].notes = this.note;
+    this.valueChanged( this.note, "notes", this.indexForNote );
```

Comment:
also here 'notes'
change all " to ' in ts files

Reply:
@SimoneMaiwald is there any particular reason of using `'` instead of `"`?

Reply:
The Angular out-of-the-box lint check wants single quotes around strings instead of double quotes and when we move to ddp-angular they have an automated lint check running …